tests/test_isolatedStarAssociation.py

```diff
@@ -186,9 +186,8 @@ def _make_simdata(self,
                     # Make one star have low s/n
                     table['normCompTophatFlux_instFlux'][0] = 1.0
 
-                df = pd.DataFrame(table)
-                df.set_index('sourceId', inplace=True)
-                data_refs.append(lsst.pipe.base.InMemoryDatasetHandle(df, storageClass="DataFrame"))
+                tbl = astropy.table.Table(table)
+                handles.append(lsst.pipe.base.InMemoryDatasetHandle(tbl, storageClass="DataFrame"))
```

Comment:
Should this be `DataFrame` or `ArrowAstropy`?

Reply:
Fixed.  But I guess nothing actually cares, given what the test actually does with the handle.